pepper-apis/src/main/resources/db-changes/schema/DDP-4552-modify-medicalprovider-street.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="20200422-modify-usermedicalprovider-street" author="sampath">
+
+        <modifyDataType tableName="user_medical_provider" columnName="street" newDataType="varchar(250)"/>
```

Comment:
Anything unusual about the street that caused this expansion? I'm wondering if we'll need to expand other tables as well, such as the `mailing_address` one.

Reply:
mailing_address contains two columns: street1 and street2 with length 100 each. we are good, no issues there.